lint-compose/src/main/kotlin/team/duckie/quackquack/lint/compose/ListNamingDetector.kt

```diff
@@ -0,0 +1,87 @@
+/*
+ * Designed and developed by 2022 SungbinLand, Team Duckie
+ *
+ * Licensed under the MIT.
+ * Please see full license: https://github.com/sungbinland/quack-quack/blob/main/LICENSE
+ */
+
+@file:Suppress(
+    "UnstableApiUsage",
+    "SameParameterValue",
+)
+
+package team.duckie.quackquack.lint.compose
+
+import com.android.tools.lint.client.api.UElementHandler
+import com.android.tools.lint.detector.api.Category
+import com.android.tools.lint.detector.api.Detector
+import com.android.tools.lint.detector.api.Implementation
+import com.android.tools.lint.detector.api.Issue
+import com.android.tools.lint.detector.api.JavaContext
+import com.android.tools.lint.detector.api.Scope
+import com.android.tools.lint.detector.api.Severity
+import com.android.tools.lint.detector.api.SourceCodeScanner
+import org.jetbrains.uast.UVariable
+import org.jetbrains.uast.kotlin.KotlinUField
+
+private const val BriefDescription = "List 네이밍 끝에는 s를 사용해야 합니다."
+private const val Explanation = "네이밍 스타일을 통일하기 위해 List 네이밍 끝에는 s를 사용해야 합니다."
+
+// List Type의 QualifiedName
+private const val ListQualifiedName = "java.util.List"
+
+val ListNamingIssue = Issue.create(
+    id = "ListNaming",
```

Comment:
```suggestion
    id = "CollectionNaming",
```

지금 생각해 보니까 List 보단 Collection 이 더 적합한 거 같네용